src/components/organisms/Contributor/ContributorCompact.tsx

```diff
@@ -34,21 +34,30 @@ interface Props<S = LocationState> {
   onSubscribe: () => void;
   onUnsubscribe: () => void;
   to?: LocationDescriptor<S>;
+  handleNameClick?: () => void;
   className?: string;
 }
 export const ContributorCompact = ({
   contributor,
   onSubscribe,
   onUnsubscribe,
+  handleNameClick,
```

Comment:
I usually find `onClick` to be more idiomatic in props, rather than `handleClick` which sounds more implementation related …

Here we could go with `onNameClick` for instance …

Reply:
<3